Makefile

```
@@ -32,6 +32,7 @@ $(call add-crd-gen,ocsv1,./pkg/apis/ocs/v1,./deploy/crds,./deploy/crds)
 	gen-release-csv \
 	gen-latest-csv \
 	gen-latest-deploy-yaml \
+	gen-latest-prometheus-rules-yamls \
```

```suggestion
	gen-latest-alerts \
```

Just so everyone can understand

---

At the end of the days, its not just alerts, it will contain rules as well. By definition, they are together called Promethues Rules, that's why prometheus-rules-yamls, as rule yamls are the ones deployed.